app/controllers/application_controller/ci_processing.rb

```diff
@@ -1840,6 +1840,49 @@ def vm_button_operation(method, display_name, partial_after_single_selection = n
     vms.count
   end
 
+  def process_cloud_object_storage_buttons
```

Comment:
can you, please, pass in `params[:pressed]` as an argument here?
(better for testing, code readability etc.)